### PR TITLE
Jetpack Sync: Sync term update action type

### PR DESCRIPTION
**Pull request by gititon**
This change indicates whether the term save was a creation or an edit/update.

#### Changes proposed in this Pull Request:

This change indicates whether the term save was a creation or an edit/update.

#### Testing instructions:

phpunit

### CODE REVIEW

File: sync/class.jetpack-sync-module-terms.php [hunk 1]
```diff
@@ -73,14 +74,29 @@ function save_term_handler( $term_id, $tt_id, $taxonomy ) {
...
+
+		if ( 'edited_term' === $current_filter ) {
+			/**
+			 * Fires when the client needs to update a term
+			 *
+			 * @since 4.2.0
+			 *
+			 * @param object the Term object
+			 */
+			do_action('jetpack_sync_save_term', $term_object);
```
Comment (enejb) [inline sync/class.jetpack-sync-module-terms.php]: minor space missing before ( and after )

File: sync/class.jetpack-sync-module-terms.php [hunk 2]
```diff
@@ -73,14 +74,29 @@ function save_term_handler( $term_id, $tt_id, $taxonomy ) {
...
+
+		if ( 'created_term' === $current_filter ) {
+			/**
+			 * Fires when the client needs to add a new term
+			 *
+			 * @since 5.0.0
+			 *
+			 * @param object the Term object
+			 */
+			do_action('jetpack_sync_add_term', $term_object);
```
Comment (enejb) [inline sync/class.jetpack-sync-module-terms.php]: here too

**Comment by lezama** [comment]
let's deploy the required WPCOM change before merging this one!

File: sync/class.jetpack-sync-module-terms.php [hunk 3]
```diff
@@ -73,8 +74,21 @@ function save_term_handler( $term_id, $tt_id, $taxonomy ) {
...
+			 * @since 5.0.0
+			 *
+			 * @param object the Term object
+			 */
+			do_action( 'jetpack_sync_add_term', $term_object );
+		}
+
 		/**
-		 * Fires when the client needs to sync a new term
+		 * Fires when the client needs to update a term
```
Comment (lezama) [inline sync/class.jetpack-sync-module-terms.php]: "Fires after updating a term" ?

File: sync/class.jetpack-sync-module-terms.php [hunk 4]
```diff
@@ -73,8 +74,21 @@ function save_term_handler( $term_id, $tt_id, $taxonomy ) {
 			$term_object = get_term_by( 'id', $term_id, $taxonomy );
 		}
 
+		$current_filter = current_filter();
+
+		if ( 'created_term' === $current_filter ) {
+			/**
+			 * Fires when the client needs to add a new term
```
Comment (lezama) [inline sync/class.jetpack-sync-module-terms.php]: "Fires after adding a new term" ?

**Pull request merged**